{"name": "Проверка получения ошибки аутентификации при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "steps": [{"name": "Отправка POST запроса на авторизацию с невалидными данными", "status": "passed", "start": 1760866186876, "stop": 1760866187243}], "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Appear.407', 'password': '6$7GX**gs@'}"}], "start": 1760866186876, "stop": 1760866187244, "uuid": "e87189a8-91c6-4046-b0b7-aa26cc676a8f", "historyId": "0595200bbe784059bc11416e1cba4518", "testCaseId": "c51f304cd3dfff9c90b10c807d062f2c", "fullName": "tests.tests_couriers.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests.tests_couriers"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Sergei"}, {"name": "thread", "value": "9908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_couriers.test_courier_login"}], "titlePath": ["tests", "tests_couriers", "test_courier_login.py", "TestCourierLogin"]}